### PR TITLE
Add Nano Stores Icon

### LINKED ISSUE #11223: Request: Nano Stores
### Brand Name

Nano Stores

### Website

https://github.com/nanostores

### Popularity Metric

Almost 5k stars & Main Repository: https://github.com/nanostores/nanostores
56890 Weekly Downloads & Active: https://www.npmjs.com/package/nanostores

### Official Resources for Icon and Color

- SVG: https://nanostores.github.io/nanostores/logo.svg
- Color: `#000000`

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by jrhaberland**
nanostores

**Issue:** closes #11223

**Popularity metric:**
Almost 5k stars & Main Repository: https://github.com/nanostores/nanostores
56890 Weekly Downloads & Active: https://www.npmjs.com/package/nanostores

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
Color: `#000000`

### CODE REVIEW

**Comment by wontory** [comment]
Thx!

File: icons/nanostores.svg
Comment (uncenter) [inline icons/nanostores.svg:1]: Seeing some misalignment. Try recreating from the original again maybe?
CleanShot 2024-09-08 at 10 25 42

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/nanostores.svg
Comment (jrhaberland) [reply icons/nanostores.svg:1]: Yeah I'll give that a shot, not sure what happened. Might need to use more precision, but I'll play around with it to see what gives the best result.

Comment (jrhaberland) [reply icons/nanostores.svg:1]: I think I accidentally used wrong precision last time. It should be better now

**Review by uncenter — APPROVED** [review]
Just waiting on a confirmation from another review that this is within our popularity metrics.

**Comment by wontory** [comment]
> 8. Where applicable, the primary repository for the brand's GitHub project meets the following requirements (see note below):
>     - A minimum of 5k GitHub stars will be required for consideration, providing,
>     - The repository is still active, and,
>     - Its [star history](https://star-history.com/) is on a consistently upward trajectory.

I'm glad this icon was added to simple icons.
This library is a state management tool recommended by [Astro framework](https://docs.astro.build/en/recipes/sharing-state/).
The repository is 5.2k star, which has been well managed until recently.

[Star History Chart](https://star-history.com/#nanostores/nanostores&Date)

**Comment by uncenter** [comment]
Thanks, but the repository is barely within our criteria for GitHubs stars (just 200 over) and below our npm downloads metric. I'll need someone else from our team to check this out first.

**Review by LitoMore — APPROVED** [review]
LGTM.

**Pull request merged**
